cluster/src/main/resources/snappydata/interpreter-setting.json

```diff
@@ -0,0 +1,70 @@
+[
```

Comment:
Can user change this file ? I guess not. Then why to keep in a Json file. Does Zeppelin mandates it ?